packages/instantsearch-react/docs/layouts/api.ejs

```diff
@@ -23,11 +23,11 @@
 
       <div class="headings">
         <ul>
-          <% for (const heading of headings) { %>
+          <% headings.forEach( heading => { %>
```

Comment:
It would be nice to understand why it fails, but no biggie, we might not even use this template later.